Source/Analytics/Web/src/assets/main.scss

```diff
@@ -1,3 +1,98 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400&display=swap');
+
+html {
+    font-size: 16px;
+    font-family: "Lato", "Helvetica Neue", sans-serif;
+}
+
 body {
-  margin: 0;
+    margin: 0;
+}
+
+$rc-red: #ca0000;
```

Comment:
these should be defined in a colors.scss that we can include in the different scss-files that use colors.